app/schema.py

```diff
@@ -25,14 +25,14 @@
 
 
 class BaseWithFetched(BaseModel):
-    fetched: datetime = datetime.now()
-    id: int
+    Fetched: str = datetime.now().isoformat()
```

Comment:
I think these attribute names should probably remain snake cased, and I'm guessing you're tempted to go with the casing that seems native to DynamoDB? If you really want the documents in DynamoDB to use UpperCase then I'd take a look at this issue: https://github.com/samuelcolvin/pydantic/issues/205

If I understand it correctly, it kind of outlines how you can automatically export a model where all the keys will be UpperCased in the conversion.